app/views/images/_index.html.erb

```diff
@@ -1,4 +1,7 @@
 <h4>Images</h4>
+<% if @map.warpables.empty? %>
```

Comment:
Would this get hidden once an image is uploaded? Thanks!

Reply:
I've added the Js for the same within https://github.com/govindjeevan/mapknitter/commit/d5366ba51ab9ee4a09ff0407146d97fb1d30f834#diff-8f5d4f54ecb0c1e4f094001ab14aa4d0